community.md

```diff
@@ -36,11 +36,23 @@ the xmonad developer team:
 
 <div class="list-col-3" markdown="1">
 
+*   [Brandon S Allbery](https://github.com/geekosaur)
+*   [Brent Yorgey](https://byorgey.wordpress.com/)
+*   [Daniel Wagner](http://www.dmwit.com/)
+*   [Sibi Prabakaran](https://psibi.in/)
+*   [slotThe](https://github.com/slotThe)
```

Comment:
@slotThe Shall I link this to GitLab, or somewhere else? You don't need to dox yourself if you don't want to, just let me know what URL you prefer.

Reply:
I decided to not dox myself at this point in time, so GitHub is totally fine (it links to the GitLab as well so there's no real difference)